tests/snappy_pipeline/workflows/test_workflows_wgs_cnv_annotation.py

```diff
@@ -95,11 +94,11 @@ def wgs_cnv_annotation_workflow(
 def test_vcf_cnv_filter_step_part_get_input_files(wgs_cnv_annotation_workflow):
     """Tests VcfCnvFilterStepPart.get_input_files()"""
     wildcards = Wildcards(
-        fromdict={"mapper": "bwa", "index_ngs_library": "P001-N1-DNA1-WGS1", "caller": "erds_sv2"}
+        fromdict={"mapper": "bwa", "index_ngs_library": "P001-N1-DNA1-WGS1", "caller": "delly2"}
     )
     # Define expected
     wgs_cnv_calling_path = (
-        "WGS_CNV_CALLING/work/bwa.erds_sv2.merge_genotypes/out/bwa.erds_sv2.merge_genotypes"
+        "WGS_CNV_CALLING/output/bwa.delly2.P001-N1-DNA1-WGS1/out/bwa.delly2.P001-N1-DNA1-WGS1"
```

Comment:
Why is only delly2 and not also gcnv added here? Both are defined in the minimal_config.

Reply:
Only `delly2` defined in the test wildcard object - single test.